### ISSUE TITLE
applet should automatically shift focus to search field when opened

### ISSUE THREAD

**Issue by PGQT** [wiiznokes/clipboard-manager#77]
#### Proposed Change
when the applet icon is clicked, there is a search field, to find previous clipboard items. It would be a big improvement in my opinion, if this field was selected and ready to type into immediately on clicking the applet icon. Since other fields that were previously active, are rendered inactive as soon as the applet is clicked, it seems that this feature, if implemented, would not cause any conflicts with existing behavior.

**Comment by wiiznokes** [comment]
Yeah, i also miss this behavior. It's currently implemented, but there is a bug somewhere. I already reported it here: https://github.com/pop-os/cosmic-panel/issues/211

**Comment by wiiznokes** [comment]
Maybe it should be reported in libcosmic instead. That would also bump it so it can be seen

**Comment by PGQT** [comment]
would you like me to submit a bug report there? or will you just copy you report over to the libcosmic repo?

**Comment by wiiznokes** [comment]
You can report is you want. Also, i noticed that is work with the horizontal layout, if you want a temporary solution

**Comment by wiiznokes** [comment]
> Maybe it should be reported in libcosmic instead

Probably a panel bug in the end
